nixos/tests/nixos-rebuild-target-host.nix

```diff
@@ -119,7 +145,7 @@ import ./make-test-python.nix ({ pkgs, ... }: {
 
       # This test also ensures that sudo is not called without --use-remote-sudo
       with subtest("Deploy to root@target"):
-        deployer.succeed("nixos-rebuild switch -I nixos-config=/root/configuration-1.nix --target-host root@target &>/dev/console")
+        deployer.succeed("nixos-rebuild switch -I nixos-config=/root/configuration-1.nix --target-host root@target --build-host builder &>/dev/console")
```

Comment:
I think it would be a good idea to separate this out into another test

Reply:
Ideally, we would have 4 tests for all combinations of `--target-host` and `--build-host` being present and not present